server/src/core/server/data/cache/userCache.ts

```diff
@@ -160,6 +160,7 @@ export class UserCache implements IDataCache {
     return {
       ...parsed,
       createdAt: new Date(parsed.createdAt),
+      lastFeaturedDate: new Date(parsed.lastFeaturedDate),
```

Comment:
Is `lastFeaturedDate` sometimes null/undefined? If so, do we want to check that and set the value accordingly?

Reply:
good point, updated, thanks!